### ISSUE TITLE
Vertical tabs component is not accessible

### ISSUE THREAD

**Issue by Angelmmiguel** [bitnami/hex#202]
## Code

* The tab button doesn't have any outline

## Documentation

* Add the `aria-live: 'polite'` property to the content
* Set the `tablist` role
* Remove `tabindex` to make all buttons clickable

**Comment by Angelmmiguel** [comment]
Closed by #203
